### DISCUSSION TITLE
How do i start Desktop environment in linux mint with x410 or some other x-server

### DISCUSSION THREAD

**Discussion by dmcodex** [sileshn/LinuxmintWSL#7] [Q&A]
is there a way to start the full desktop like other distros such as ubuntu 
when i try to start a cinnamon session it says x11 is not supported 
please could you help 

thanks

**Comment by sileshn** [comment]
This is barebones. You need to install a desktop environment and a display manager. I haven't tried it, but it should work in theory.

**Comment by JOs86** [comment]
FYI: (For example) XFCE4 works, (at least it seems so..  need more time to check it properly)...

[Please, correct me if wrong/ inaccurate]

sudo apt install xfce4, (then you need to start it: "sudo startxfce4")... 
then
$ ifconfig
(and then ipconfig in powershell) ...
You will need to edit .bashrc (and use correct IP address there, i.e. IPv4):
nano .bashrc
   export DISPLAY=XXX.XXX.XXX.XX:X


Also, you might need to edit wslconfig, adding these lines might work:
[wsl2]
guiApplications=false

You can use RDP or VcXSrv X server for Windows, etc. (check the documentation, there might be some limitations).

Info (for example) here:
https://autoize.com/xfce4-desktop-environment-and-x-server-for-ubuntu-on-wsl-2/
https://forum.xfce.org/viewtopic.php?id=16906
